contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -231,34 +237,54 @@ contract SimpleInterestTermsContract is TermsContract {
         // token registry.
         bytes32 principalTokenIndexShifted =
             parameters & 0xff00000000000000000000000000000000000000000000000000000000000000;
```

Comment:
Let's add a comment here that specifies that we reserve the bottom 108 bits for collateralization-specific parameters, if they're relevant.  i.e., copying from Collateralized.sol:

```
     * For collateralized terms contracts, we reserve the lowest order 108 bits
     * of the terms contract parameters for parameters relevant to collateralization.
```